setup.py

```diff
@@ -20,6 +20,7 @@
     "pyyaml",
     "pandas",
     "glymur",
+    "scikit-learn=0.23.2",
```

Comment:
Please change "scikit-learn=0.23.2" to "scikit-learn==0.23.2"

Reply:
Fixed in [this commit](https://github.com/TIA-Lab/tiatoolbox/pull/45/commits/6b457bf90bc7be48c8eb714edadd672c28066ea1)